recipes/cofold/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# ViennaRNA does not support Python 3 yet.
+./configure --prefix=$PREFIX --without-perl -q
+make
```

Comment:
Hi Björn,

It worked, thanks :)
I'll do a new pull request tomorrow to replace the old one.

Cheers,
Hassan

On Wednesday, 20 July 2016, yhoogstrate <notifications@github.com> wrote:

> In recipes/cofold/build.sh
> <https://github.com/bioconda/bioconda-recipes/pull/1774#discussion_r71528081>
> :
>
> > @@ -0,0 +1,5 @@
> > +#!/bin/sh
> > +# ViennaRNA does not support Python 3 yet.
> > +./configure --prefix=$PREFIX --without-perl -q
> > +make
>
> The package contains some precompiled libraries (libRNA.a) compiled with
> gcc<2.7. You can easiliy get rid of it by running:
>
> ./configure etc etc
> make clean
> make
> make install
>
> Maybe this solves your travis issue?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bioconda/bioconda-recipes/pull/1774/files/2b4b32f57a4541c3ccc165773d62f6b76ae5734d#r71528081>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALpc4m9PXbdCWRFCTgeCcxL-ZsSIVYxqks5qXihlgaJpZM4I3_GB>
> .
>